addon/utils/attach-dropdown.ts

```diff
@@ -65,6 +64,20 @@ export default function attachDropdown(
     hide()
   ];
 
+  if (options.placementStrategy === 'auto') {
+    middlewares.push(
+      autoPlacement({
+        allowedPlacements: ['top', 'bottom']
+      })
+    );
+  } else {
+    middlewares.push(
+      flip({
+        fallbackPlacements: ['top', 'bottom']
+      })
+    );
+  }
```

Comment:
This is to not break the other use cases of this util. For example in the search tutorialisation popover where it's sometimes on a side, sometimes below

Reply:
https://floating-ui.com/docs/autoplacement#conflict-with-flip